GVFS/FastFetch/NativeWindowsMethods.cs

```diff
@@ -0,0 +1,121 @@
+﻿using GVFS.Common.Tracing;
+using Microsoft.Win32.SafeHandles;
+using System;
+using System.ComponentModel;
+using System.IO;
+using System.IO.MemoryMappedFiles;
+using System.Runtime.InteropServices;
+
+namespace FastFetch
+{
+    public class NativeWindowsMethods
+    {
+        private const int AccessDeniedWin32Error = 5;
+
+        public static unsafe void WriteFile(ITracer tracer, byte* originalData, long originalSize, string destination, ushort mode /* ignored on windows */)

```

Comment:
Since there's no inheritance relationship now, can the `mode` parameter be dropped from this method?